### ISSUE TITLE
Is Linux supported?

### ISSUE THREAD

**Issue by maffblaster** [sakurity/securelogin#7]
Yo! The [blog post ](https://medium.com/@homakov/securelogin-forget-about-passwords-c1bf7b47f698) mentions Linux support. The README's section on Linux is blank. Maybe I'm missing something. Is Linux actually supported (yet)?

**Comment by homakov** [comment]
Yes it's supported and will be distributed as .AppImage (is it ok? or .deb?), there's a little issue with registering system wide protocol for gvfs-open, when it will be sorted out will update this issue.

Maybe you could help out with gentoo? Electron does not seem to deal with system protocols for Linux.

**Comment by ordian** [comment]
Consider using [flatpak](http://flatpak.org/getting.html) for linux. 
It has unofficial gentoo support and there is [electron-installer-flatpak](https://www.npmjs.com/package/electron-installer-flatpak).

**Comment by probonopd** [comment]
Providing an [AppImage](http://appimage.org/) would have, among others, these advantages:
- Applications packaged as an AppImage can run on many distributions (including Ubuntu, Fedora, openSUSE, CentOS, elementaryOS, Linux Mint, and others)
- One app = one file = super simple for users: just download one AppImage file, [make it executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80), and run
- No unpacking or installation necessary
- No root needed
- No system libraries changed
- Works out of the box, no installation of runtimes needed
- Optional desktop integration with `appimaged`
- Optional binary delta updates, e.g., for continuous builds (only download the binary diff) using AppImageUpdate
- Can optionally GPG2-sign your AppImages (inside the file)
- Works on Live ISOs
- Can use the same AppImages when dual-booting multiple distributions

[Here is an overview](https://github.com/probonopd/AppImageKit/wiki/AppImages) of projects that are already distributing upstream-provided, official AppImages.

__electron-builder__ can generate an AppImage automatically for you.

**Comment by homakov** [comment]
There is appimage that is auto generated by electron-builder https://securelogin.pw/SecureLogin-1.0.0-x86_64.AppImage However I never tested if it properly works because... priorities. But eventually will.

**Comment by ordian** [comment]
The link is broken (404).

**Comment by homakov** [comment]
Oh, i was searching for right one and when found it download started & it wasn't saved in chrome's address bar. So here's right one https://securelogin.pw/apps/SecureLogin-1.0.0-x86_64.AppImage
